### PR TITLE
Overhauled Ubuntu install steps via new page

### PR DESCRIPTION
**Pull request by cwarnermm**
This PR also:
- adds links for the new Ubuntu install page to the navigation pane and the deployment landing page for easy access
- removes the Ubuntu/Debian tab from the deployment landing page

@nab-77 - Please update this PR to indicate where the Marketing CTA links should be added. Thanks!

Note: The docs link via ``mattermost.com/download`` to the Ubuntu install docs will need to be updated from ``https://docs.mattermost.com/install/installing-ubuntu-2004-LTS.html`` to ``https://docs.mattermost.com/install/install-ubuntu.html``. Docs will create client-side redirects to ensure users with legacy Ubuntu install URLs are redirected to the correct page.

@emdecr - This PR is now ready for you to add the Marketing CTAs and code formatting. Thank you! Can we get your help with the note above too to update the website URL?

### CODE REVIEW

File: source/install/install-ubuntu.rst [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    - If you are running the Mattermost Server and database a single system, we recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ as this greatly reduces setup and ongoing maintenance.
+
+Add the Mattermost Server PPA repository
+----------------------------------------
+
+In a terminal window, run the following command to add the Mattermost Server repositories:
+
+.. code-block:: none
+
+    curl -o- https://deb.packages.mattermost.com/repo-setup.sh | sudo bash -s mattermost
```
Comment (nab-77) [inline source/install/install-ubuntu.rst:51]: @emdecr this is a cmd for the copy-action button

File: source/install/install-ubuntu.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+.. code-block:: none
+
+    sudo apt update
+
+After any updates (and any system reboots) are complete, installing the Mattermost Server is now a single command:
+
+.. code-block:: none
+
+    sudo apt install mattermost -y
```
Comment (nab-77) [inline source/install/install-ubuntu.rst:66]: @emdecr this is the cmd for the copy-action button

**Review by nab-77 — COMMENTED** [review]
Added copy-action items

File: source/install/install-ubuntu.rst [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    </div>
+
+  </div>
+
+You can install the Mattermost Server using our ``.deb`` signed packages using the Mattermost PPA (Personal Package Archive). Deployment includes 4 steps: `add the PPA repository <#add-the-mattermost-server-ppa-repository>`__, `install <#install>`__, `setup <#setup>`__, and `update <#updates>`__.
+
+.. important::
+    
+    - Using the Mattermost PPA not only provides the quickest way to install a Mattermost Server, but also provides automatic updates. Unlike the Mattermost Omnibus, this install method is used for both single and clustered installations. 
+    - If you are running the Mattermost Server and database a single system, we recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ as this greatly reduces setup and ongoing maintenance.
```
Comment (nab-77) [inline source/install/install-ubuntu.rst:42]:
```suggestion
    - If you are running the Mattermost Server and database on a single system, we recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ as this greatly reduces setup and ongoing maintenance.
```

File: source/install/install-ubuntu.rst [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    sudo systemctl enable mattermost.service
+
+Why does Mattermost fail to start at system boot?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your database is on the same system as your Mattermost Server, we recommend editing the default ``/lib/systemd/system/mattermost.service`` systemd unit file to add ``After=postgresql.service`` and ``BindsTo=postgresql.service`` to the ``[Unit]`` section.
+
+.. note::
+	
+	We recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ over the ``deb`` signed package if you are running the Mattermost Server and database a single system as this greatly reduces setup and ongoing maintenance.
```
Comment (nab-77) [inline source/install/install-ubuntu.rst:140]:
```suggestion
	We recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ over the ``deb`` signed package if you are running the Mattermost Server and database on a single system as this greatly reduces setup and ongoing maintenance.
```

**Review by nab-77 — CHANGES_REQUESTED** [review]

**Review by nab-77 — APPROVED** [review]
Just needs the formatting/tracking from @emdecr

**Comment by emdecr** [comment]
Missed the tag on this one; for some reason I'm not getting notifications for this PR via the GitHub <> MM integration 😞 

@cwarnermm I took a stab at the merge conflict in [deployment.rst](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6358/guides/deployment.html) that was happening when I pulled from master, but worth a double-check to make sure that's all still good. Sorry if I mucked things up 😅 

Lemme know, if anything - TY!

**Review by emdecr — APPROVED** [review]

**Comment by emdecr** [comment]
Oh and yes, once we merge, happy to change the URL on mm.com/download – just keep me in the loop or ping me, and I'll make the change.

**Pull request merged**